lib/source-destination/source-destination.ts

```diff
@@ -14,13 +14,48 @@
  * limitations under the License.
  */
 
+import { Transform as TransformStream } from 'stream'
 import { EventEmitter } from 'events';
 import { ReadResult, WriteResult } from 'file-disk';
 import * as fileType from 'file-type';
 import { getPartitions, GetPartitionsResult } from 'partitioninfo';
 import { extname } from 'path';
-import { arch } from 'process';
-import { Stream as HashStream } from 'xxhash';
+import { XXHash3 } from 'xxhash-addon';
+import { inherits } from 'util'
+
+// @ts-ignore
+function HashStream(seed, outEnc) {
```

Comment:
You can create an actual class with `class HashStream extends Transform` and avoid inherits. It will probably remove the need for all the `@ts-ignore` comments.